modules/telco-core-yaml-ref-networking.adoc

```diff
@@ -0,0 +1,127 @@
+// Module included in the following assemblies:
+//
+// *
+
+:_content-type: REFERENCE
+[id="networking-yaml_{context}"]
+= Networking reference YAML
+
+[id="telco-core-network-yaml"]
+.Network.yaml
+[source,yaml]
+----
+include::snippets/telco-core_Network.yaml[]
```

Comment:
Please make all your filenames lowercase.

Reply:
These files are pulled from a dev repo and are just prefixed for the final include. I can update the script to lowercase the files, but my preference would be to include the files as is for ease of comparison with the Dev source. These funky file names do not get surfaced in the URL FYI.